src/main/java/com/impossibl/postgres/jdbc/PGResultSet.java

```diff
@@ -2055,9 +2061,15 @@ int move(String type, Object loc) throws SQLException {
 
     result = null;
 
-    String sql = String.format("MOVE %s %s IN %s", type, loc, cursorName);
+    StringBuilder sb = new StringBuilder();
```

Comment:
If this change is primarily about *performance*, the default ``StringBuilder()`` constructor initializes with an initial capacity of 16 characters.  The string here is basically 16 characters *before* the parameters are inserted.  We don't want to waste space, but at the same time we don't want to immediately have to expand the capacity either.  ``StringBuilder`` will double the capacity when it grows, so it is probably safe to start out with ``new StringBuilder(32)`` at a minimum, or probably safer, ``new StringBuilder(48)``.